.github/workflows/publish.yml

```diff
@@ -15,13 +18,34 @@ jobs:
         uses: actions/checkout@v3
       - name: Build Website
         run: |
-          docker run -v ${PWD}/sources:/hugo/content/docs/grafana-cloud --rm grafana/docs-base:latest /bin/bash -c 'make hugo'
+          docker run -v ${PWD}/sources:/hugo/content/docs/agent --rm grafana/docs-base:latest /bin/bash -c 'make hugo'
   sync:
     runs-on: ubuntu-latest
     needs: test
     steps:
-    - uses: actions/checkout@v3
-    - run: git clone --single-branch --no-tags --depth 1 -b master https://grafanabot:${{ secrets.GH_BOT_ACCESS_TOKEN }}@github.com/grafana/website-sync ./.github/actions/website-sync
+
+    - name: Checkout Agent repo
+      uses: actions/checkout@v3
+
+    - name: Checkout Actions library
+      uses: actions/checkout@v3
+      with:
+        repository: "grafana/grafana-github-actions"
+        path: ./actions
+        ref: docstarget
```

Comment:
Is this right? I looked at the repo and it should be the main branch at the `./docs-target` path.

Reply:
Good catch, thanks. This remains from when testing from the mdisibio/grafana-github-actions fork. Fixing it now.